models/config/binance_parser.py

```diff
@@ -140,4 +140,7 @@ def parser(app, binance_config, args={}):
                 app.granularity = to_internal_granularity(config['granularity'])
                 app.smart_switch = 0
             else:
-                raise ValueError('granularity supplied is not supported.')
\ No newline at end of file
+                app.granularity = int(config['granularity'])
+                app.smart_switch = 0
```

Comment:
this part we need to talk about and test. when I did tests with coinbase with the `raise` present, it worked. with binance it went into the exception although it was specified correctly.

I think we should have 4 eyes on this.